src/components/pipeline-overview/PipelineRunItem.scss

```diff
@@ -0,0 +1,5 @@
+.odc-pipeline-run-item {
```

Comment:
Can we update the `odc-` prefix for class name and use what we are using for console-plugin? Update all the ODC class name in this PR.

Reply:
Updated the PR. PTAL.